### PR TITLE
first-capstone-project

### PR DESCRIPTION
**Pull request by Masouma96**
Made the changes required.
Added video recording link to readme file.
In this capstone project I used html , css ,javascript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,228 +1,188 @@
...
+
+      <section class="partner1" id="join-section">
+        <div class="partner1-img">
+          <h1>partner</h1>
+
+          <a href="#">mazila</a><a href="#">Google</a><a href="#">NAVER</a>
+          <a href="#">airbnd</a>
+          <a href="#">daumkakao</a>
+        </div>
+      </section>
```
Comment (alexander16108) [inline index.html:169]: In this  section, your partners logos should be  aligned to the centre 

your design
image

required design
image

File: index.html [hunk 2]
```diff
@@ -1,228 +1,188 @@
...
+        <div class="right">
+          <p>BAKHTAR Migration INC.some rights reserved</p>
+        <p class="p2">
+           Address: No. 15,Barchee St.,
+          13 District, Kabul City , Afghanistan, If you want to contact us for
+          official business, your call will be transferred to the appropriate
+          department.
+        </p>
+        </div>
+      </section>
```
Comment (alexander16108) [inline index.html:185]: in the footer section of you page, there should be more margin around the contents of the footer and the body `justify-content: center` could if fix this and more padding between the logo and the address

your design 
image

required design
image

File: index.html [hunk 3]
```diff
@@ -1,228 +1,188 @@
...
+        <img src="images/menu.svg" alt="" class="menu" />
+        <img src="images/logo1.jpg" alt="" class="desk-logo" />
+        <div class="nav-links">
+          <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#" class="bold">BAKHTAR INC</a></li>
```
Comment (alexander16108) [inline index.html]: please create a link that will take you to the about section on your desktop version
your design
image

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @masouma2020 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Hi I can see u really try but there is a good bit that is still need to be done 👍🏾 

## Highlights
- passing linters ✔️ 
- video recording ❌ 


## Required Changes ♻️

- You should record a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
-Your video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds attractive or is proud of making.
- your README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
-  The project is deployed properly, all pages, links, and images work as expected.
-  A link to the deployed project is present in the README.
- You should make use of GitHub Pages to deploy the web pages.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Masouma96** [comment]
Hi sir
I made the changes.
Please see it.
Thank you

**Comment by m05e5** [comment]
Can you fix the stylelint issue first?

**Comment by Masouma96** [comment]
Yes, I can.

**Comment by Masouma96** [comment]
The error message is not clear to me I do not know how to fix it

**Comment by m05e5** [comment]
Just remove line 736 in your style.css since its an extra empty line

**Comment by Masouma96** [comment]
But there is no extra line

**Comment by m05e5** [comment]
You should put an empty line here 
image

**Comment by Masouma96** [comment]
Can you check noe please.

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @masouma2020  🔥 

Good job on implementing the required changes. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Masouma96** [comment]
Awesome, thank you so much!

**Pull request merged**